quotation_template_automatic_description/models/sale_order_template.py

```diff
@@ -0,0 +1,10 @@
+from odoo import api, fields, models, _
+
+
+class SaleOrderTemplate(models.Model):
+    _inherit = "sale.order.template"
+
+    auto_update_description = fields.Boolean(
```

Comment:
@sswapnesh this is wrong. Look at the specs of 1042 again please, it has to be on line level, not template level.